### ISSUE TITLE
Any plan to implement other RPC

### ISSUE THREAD

**Issue by pbmean-dh** [sonic-net/sonic-netconf-server#10]
Hello. 

First of all, thank you for your excellent efforts in implementing NETCONF for SONiC.

I am developer to try to apply NETCONF in SONiC device and currently reviewing and testing the NETCONF functionality you have developed.
Do you have any plans to implement other RPCs like EDIT-CONFIG, in addition to GET and GET-SCHEMA?
If so, could you share the development schedule you are considering?

Wishing you a wonderful day as always.

Thank you.

**Comment by hussamsh** [comment]
I have already implemented a good chunk of the NETCONF RFC (EDIT-CONFIG, LOCKING, COMMIT ... etc) but will hold off on sharing these changes until I make sure that this initial implementation is working/building correctly with the current version of SONiC.

**Comment by pbmean-dh** [comment]
Wow!  That's good news.  I hope your great works will be merged into master branch of SONiC.

**Comment by knightswim** [comment]
Looking forward to sharing the full version，Do you have a specific plan @hussamsh

**Comment by hussamsh** [comment]
@knightswim  Pull request for the NETCONF submodule is ready on the build-image repo, awaiting approval from maintainers. I will be sending them an email this week to try to expedite things. 
On my side the full version will be released once this initial version is merged

**Comment by rijnma** [comment]
We’re exploring adding a Netconf interface to SONiC by means of your project.  I see you’ve made headway implementing new features. We require a candidates database.  The candidate configuration needs to be written to the running datastore, followed by a confirmation of its success. I find the last part very interesting: what happens when something goes wrong during writing. Please correct me if I’m incorrect, but this seems to be part of your unshared code. Is this correct? Provided this is correct, can you share this new feature to test?

**Comment by sugisono-omw** [comment]
We are looking forward to releasing additional operations including edit-config.
Our concern is it takes longer to release the full version of NETCONF module since you told there was a plan for it.
We would appreciate it if you could release the full version in advance.
